source/shared/core_stmt.cpp

```diff
@@ -1014,6 +1034,9 @@ void core_sqlsrv_next_result( sqlsrv_stmt* stmt TSRMLS_DC, bool finalize_output_
 
         close_active_stream( stmt TSRMLS_CC );
 
+        //Clear column sql types and sql display sizes.
+        zend_hash_clean( Z_ARRVAL( stmt->col_cache ));	    
+	    
```

Comment:
Where does the cache get reinitialized after core_sqlsrv_next_result? The only initialization I see for the col_cache is in the sqlsrv_stmt constructor.